templates/forward.erb

```diff
@@ -5,5 +5,6 @@ forward-zone:
   forward-addr: <%= addr %>
 <% end -%>
 <% elsif @address != '' -%>
-  forward-addr: <%= @address -%>
+  forward-addr: <%= @address %>
 <% end -%>
+  forward-first: <%= @forward_first -%>
```

Comment:
Could we only include this line if the value of `@forward_first` is not the default?  This is really just for cleaner configs.